src/main/java/net/masterthought/cucumber/generators/TagsOverviewPage.java

```diff
@@ -38,14 +36,14 @@ public void prepareReport() {
         context.put("chart_data", generateTagValues(filterExcludedTags(tags)));
     }
 
-    static String generateTagLabels(List<TagObject> tagsObjectList) {
+    static String[] generateTagLabels(List<TagObject> tagsObjectList) {
         int tagCount = tagsObjectList.size();
         String[] tagNames = new String[tagCount];
 
         for (int i = 0; i < tagCount; i++) {
-            tagNames[i] = StringUtils.wrap(tagsObjectList.get(i).getName(), "\"");
+            tagNames[i] = tagsObjectList.get(i).getName();
```

Comment:
so now you have some logic here and also in macro.js.vm file what is even worse than now :(